src/screens/auth/Login.jsx

```diff
@@ -0,0 +1,173 @@
+import React, { useState } from "react";
+import { TextField, Box, Typography } from "@mui/material";
+import { validateInput } from "../../utils/validator";
+function Login() {
+  const [input, setInput] = useState("");
```

Comment:
make the state name meaningfully 

refer this : https://www.itglue.com/blog/naming-conventions-examples-formats-best-practices/